modules/tensor_mechanics/include/userobjects/TensorMechanicsPlasticModel.h

```diff
@@ -248,7 +248,14 @@ class TensorMechanicsPlasticModel : public GeneralUserObject
   virtual RankFourTensor consistentTangentOperator(const RankTwoTensor & trial_stress, const RankTwoTensor & stress, const Real & intnl,
                                                    const RankFourTensor & E_ijkl, const std::vector<Real> & cumulative_pm) const;
 
-  bool KuhnTuckerSingleSurface(const Real & yf, const Real & dpm) const;
+  /**
+   * Returns true if the Kuhn-Tucker conditions for the
+   * single surface are satisfied.
+   * @param yf Yield function value
+   * @param dpm plastic multiplier
+   * @param dpm_tol tolerance on plastic multiplier: viz dpm>-dpm_tol means "dpm is non-negative"
+   */
+  bool KuhnTuckerSingleSurface(const Real & yf, const Real & dpm, const Real & dpm_tol) const;
```

Comment:
`const Real &` can be replaced with just `Real` for improved readability. They are almost the same thing (except for constness within the function scope, but we do not care about this _anywhere_ in MOOSE).